js_modules/dagit/packages/core/src/partitions/DimensionRangeWizard.tsx

```diff
@@ -52,9 +52,11 @@ export const DimensionRangeWizard: React.FC<{
     }
     didSetInitialPartition.current = true;
     const query = qs.parse(window.location.search, {ignoreQueryPrefix: true});
-    const partition = query.partition as string;
-    if (partition && partitionKeys.includes(partition)) {
-      setSelected([partition]);
+    const partitions = ((query.partition as string) || '').split('|') as string[];
+    for (const partition of partitions) {
+      if (partition && partitionKeys.includes(partition)) {
+        setSelected([partition]);
+      }
```

Comment:
Hmmm, this seems a bit weird to do in the range wizard itself -- in a multi dimensional asset, there will actually be two of these in the modal and the "partitions" in this component are the per-dimension keys, and the component doesn't have enough context to know which oneto look at.

It's valid to have a multi-partitioned asset where the two partition spaces share the same keys, like `TN|TN`, `TN|VA`, `VA|TN` etc. (It'd be weird, but we don't have rules against it)

I think that `usePartitionDimensionSelections` should actually already be pre-selecting the correct initial values from the context on the parent page, but it uses the selected range and not the selected individual partition you're viewing.

Maybe implementing this PR as by changing the selections hook to read different query parameters would b e better?: https://github.com/dagster-io/dagster/blob/master/js_modules/dagit/packages/core/src/assets/usePartitionDimensionSelections.tsx#L64:L64

I think this modal is already synced to the state of the partitions page, but it's synced to the time range selector at the top of the page, not to the exact selected partition. I think the code in that hook just needs to look at the current selection as well / instead.